local/gcs.go

```diff
@@ -143,6 +143,7 @@ func handleGet(w http.ResponseWriter, r *http.Request) {
 
 // handler is the root http request handler.
 func handler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Access-Control-Allow-Origin", "http://localhost:9000")
```

Comment:
Is this needed, as this hardcodes 9000 ?

Reply:
Yeah, otherwise errors are very difficult to debug (as was my case while making this PR). It's not possible to control this port anyway via butler.py CLI interface, so keeping this.